src/python/foglamp_daemon.py

```diff
@@ -58,6 +58,47 @@ def safe_makedirs(directory):
         if not os.path.exists(directory):
             raise e
 
+def find_process_start_time(process_name):
+    """
+    Find Process start time
+
+    :param process_name: name of the process
+    :return: string start_time or None if no such process exists
+    """
+    f_ps = subprocess.Popen("ps h -C " + process_name + " -o lstart", shell=True, stdout=subprocess.PIPE)
+    start_time = f_ps.stdout.read().decode("utf-8")[:-1] # remove last \n character
```

Comment:
Yupe, that was my initial plan too. However this is linux specific, and does not work on darwin (mac). So we are using psutil py package to make life easier and that will allow to scale monitoring with ease.